docs/user/build/maven/index.rst

```diff
@@ -12,7 +12,7 @@ Using Maven
    snapshots
    javadocs
    eclipse
-   pom
+   maven_pom
```

Comment:
Trying out InttelliJ and it was failing on file `pom.rst`. This has never been a problem before..